app/assets/javascripts/mirador_widget_admin.js

```diff
@@ -60,6 +61,13 @@
       });
     }
 
+    // Setup function that listens to when an external IIIF manifest cannot be fetched
+    function setupManifestErrorListener(block) {
+      block.on('manifest-error', function(e, eventObject) {
+        manifestError(block, eventObject.error);
```

Comment:
Can we use `eventObject.block` in the call to `manifestError`?  I'm not 100% sure if it'll cause any issues, but it'll be more consistent w/ the other usage around this code.